### ISSUE TITLE
New-AzureRmResource for deployments has duplicate ResourceName switches

### ISSUE THREAD

**Issue by davidebbo** [projectkudu/ARMExplorer#117]
e.g.

```
New-AzureRmResource -ResourceName $ResourceName -Location $ResourceLocation -PropertyObject $PropertiesObject -ResourceGroupName Default-Web-NorthEurope -ResourceType Microsoft.Web/sites/deployments -ResourceName MySite -ApiVersion 2015-08-01 -Force
```

**Comment by initparam** [comment]
The same applies to the "Powershell" tab on Azure RM Explorer

**Comment by davidebbo** [comment]
Actually, that's what this issue is referring to :)

**Comment by davidebbo** [comment]
This also happens for DataLake Store (#130)

**Comment by bala16** [comment]
fixed by 391f2170c9178892fe4916ec90e390fb01e76344
